tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -199,6 +202,12 @@ public:
       void Read(NTupleSize_t globalIndex) { fField->Read(globalIndex, fObjPtr.get()); }
       void Read(RClusterIndex clusterIndex) { fField->Read(clusterIndex, fObjPtr.get()); }
       void Bind(std::shared_ptr<void> objPtr) { fObjPtr = objPtr; }
+      void BindRawPtr(void *rawPtr)
+      {
+         if (!fRawPtrCtrlBlock)
+            fRawPtrCtrlBlock = std::shared_ptr<void>(this, [](void *) {});
```

Comment:
Do we need this? In my tests, it was fine to alias an empty `std::shared_pointer`...

Reply:
That's probably right. It is a bit confusing that in this way we create non-empty shared pointer with `use_count() == 0`. Perhaps that's exactly what it is though.